web-server/v0.4/src/routes/Dashboard/RunComparison.js

```diff
@@ -686,15 +736,22 @@ export default class RunComparison extends ReactJS.Component {
         <br />
         <Card title="Cluster Tables">
           {Object.keys(tableData).map(table => (
-            <Table
-              bordered
-              id={table}
-              title={() => <h5>{table}</h5>}
-              columns={columns}
-              dataSource={tableData[table]}
-              expandedRowRender={expandedRowRender}
-              pagination={{ pageSize: 20 }}
-            />
+            <div id={'table' + table}>
+              <input
+                type="checkbox"
```

Comment:
So will this be displayed as an actual checkbox or will it be displayed as a simple button that can be pressed?